pkgs/http/lib/src/utils.dart

```diff
@@ -68,3 +68,22 @@ Stream<T> onDone<T>(Stream<T> stream, void Function() onDone) =>
       sink.close();
       onDone();
     }));
+
+/// Caching modes for any http request in a browser.
+enum CacheMode {
```

Comment:
Where is this used?

Reply:
We can use it if the user needs to use the caching modes with existing utilities instead of writing everything as String and pass it.

Reply:
@brianquinlan I guess I would move with Kevin for making those cache modes available instead of allowing the feature to make it pass any String.

Reply:
OK. Please move this to `browser_client.dart`